src/Microsoft.VisualStudio.SlnGen/Program.cs

```diff
@@ -137,7 +137,7 @@ internal static int Execute(ProgramArguments arguments, IConsole console)
             };
 
             using (ProjectCollection projectCollection = new ProjectCollection(
-                globalProperties: null,
+                globalProperties: arguments.GetGlobalProperties(),
```

Comment:
Actually this is the wrong place, I was just looking into what's going on.  Please leave this as `null` but pass `arguments.GetGlobalProperties()` to `LoadProjects()` on line 171

Reply:
thanks, done.